### PR TITLE
add(tools): Tailwindhelper

### PR DESCRIPTION
**Pull request by felixmacaspac**
| Name                 | Link                 |
| -------------------- | -------------------- |
|Tailwindhelper | https://www.tailwindhelper.com/ |

A website that helps you visualize Tailwind CSS classes and also has a unit converter tool.
___

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by basarozcan** [comment]
That domain name is NOT in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)

**Comment by joshmanders** [comment]
Hey @felixmacaspac while I feel it doesn't fall in line with the guidelines either, it's a cool and useful site. I've reached out to Adam on what he thinks and I'll let ya know. If he would like it to follow guidelines I'd suggest getting a different domain and rebranding, but I'd love to have this as an added entry in the repo!

File: README.md [hunk 1]
```diff
@@ -198,6 +198,7 @@
 - 🌍 [tailwind.run](https://tailwind.run) - Tailwind CSS fiddle with built-time features (online).
 - 🌍 [tailzilla.app](https://tailzilla.app) - Online playground for Tailwind CSS.
 - 🌍 [Flowrift](https://flowrift.com) - Beautifully designed Tailwind CSS UI blocks.
+- 🔼🌍🔧 [Tailwindhelper](https://www.tailwindhelper.com/) - Visualize Tailwind CSS classes and unit converter.
```
Comment (joshmanders) [inline README.md:201]: Awaiting feedback from Adam about the domain following guidelines, but I'd like to see it rebranded and updated. We can merge it in after that :)

**Review by joshmanders — CHANGES_REQUESTED** [review]

**Review by joshmanders — APPROVED** [review]
Adam has said this is fine as it's just a free helpful thing, so if you do decide to monetize it in any way, you'll be in violation of the guidelines. So be sure to be aware of that, as it'll be my job to make sure we don't support violations in this repo.

**Comment by felixmacaspac** [comment]
Hi @joshmanders thanks for including it in the repo, **BUT** this is not my own work nor affiliated with it. I just found this open source tool on GitHub and was using it frequently, I just thought it was an incredible tool which is why I added it here. Would there be any problem if I'm not the owner of this tool? Sorry, I didn't expect that it would be approved because I also just realized that recently.

**Comment by joshmanders** [comment]
@felixmacaspac oh nah don't sweat it, contributions are allowed from anyone, we just kind of assumed it was yours because most people who are submitting are doing so to share something they made. I'll reach out to the dev!

**Comment by joshmanders** [comment]
Shared with the creator: https://github.com/Seb-GRAF/tailwind-helper/issues/2

**Pull request merged**